src/ray/raylet/worker_pool.h

```diff
@@ -121,6 +121,10 @@ class WorkerPool {
   /// \param task_id The task that we want to query.
   bool HasPendingWorkerForTask(const Language &language, const TaskID &task_id);
 
+  /// Get the set of active object IDs from all workers in the worker pool.
+  /// \return A set containing the active object IDs.
+  std::unordered_set<ObjectID> GetActiveObjectIDs() const;
```

Comment:
this set might be big, how about we put it as a reference arg instead of a return value to avoid a copy?

Reply:
This will be optimized away:
https://shaharmike.com/cpp/rvo/